composer.json

```diff
@@ -24,7 +24,7 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": "^5.5.9 || ^7.0",
```

Comment:
Rest of the constraints use `^5.5.9|^7.0` format, please don't introduce different one.

Reply:
Ah, I keep forgetting which format is the one we use. Sorry.